include/bluetooth/gap.h

```diff
@@ -190,6 +192,7 @@ enum {
 	BT_GAP_SCA_0_20 = 7,
 };
 
+
```

Comment:
```suggestion
```

Reply:
Removed

Reply:
You might have forgotten to push that change :)

Reply:
Ah, I didn't save it. Sorry